dags/multipod/maxtext_end_to_end.py

```diff
@@ -48,7 +48,7 @@
           tpu_cores=8,
           tpu_zone=Zone.US_CENTRAL2_B.value,
           time_out_in_min=60,
-          test_name=f"{test_name_prefix}-stable-{test_script}",
```

Comment:
@jonb377 wdyt of this change ? I think in future the values of the dict are going to be paths so having keys in the test_name should be cleaner